cmd/maya-apiserver/app/server/volume_endpoint.go

```diff
@@ -87,23 +88,23 @@ func (s *HTTPServer) vsmList(resp http.ResponseWriter, req *http.Request) (inter
 		return nil, err
 	}
 
-	fmt.Println("[DEBUG] Processed Volume list request successfully")
+	glog.Infof("Processed Volume list request successfully")
 
 	return l, nil
 }
 
-// vsmRead is the http handler that fetches the details of a VSM
-func (s *HTTPServer) vsmRead(resp http.ResponseWriter, req *http.Request, vsmName string) (*v1.Volume, error) {
+// VolumeRead is the http handler that fetches the details of a Volume
+func (s *HTTPServer) VolumeRead(resp http.ResponseWriter, req *http.Request, volName string) (*v1.Volume, error) {
 
-	fmt.Println("[DEBUG] Processing Volume read request")
+	glog.Infof("Processing Volume read request")
 
-	if vsmName == "" {
+	if volName == "" {
 		return nil, CodedError(400, fmt.Sprintf("Volume name is missing"))
```

Comment:
I think you should log this error, Do you? There are other places like this!

Reply:
`CodedError` accept only string right now as a arg, and glog don't have any error method which return string.